examples/app.py

```diff
@@ -87,17 +88,12 @@
    $ flask -a app.py run
 
 
-9. Login into the application. Open in a web browser
-`http://localhost:5000/login/ and use the user previously created
-(user: info@inveniosoftware.org).
-
-
-10. Open a web browser and enter to the url
+9. Open a web browser and enter to the url
 `http://localhost:5000/records/RECORD_PID/preview` where
 `RECORD_ID` is a number between 1 and 10.
 
 
-11. Open now a record that contains several files (The last record created).
+10. Open now a record that contains several files (The last record created).
 By default, it is showing the first document, but you can set another file
 using a query string like
 `http://localhost:5000/records/6/preview?filename=csvfile.csv`
```

Comment:
Because of the endpoint that has been declared here: https://github.com/inveniosoftware/invenio-previewer/pull/40/files#diff-86d2f29b7151f1dfb6b158f5f495ba35R147, you don't need to use the querystring anymore. This line should be something like: ``http://localhost:5000/records/6/preview/csvfile.csv``. Reprhrase also the last sentence of the comment with something like: ``(...) but you can see another file adding the filename, like: `` ;)